snp_motion_planning/src/planner_profiles.hpp

```diff
@@ -170,7 +170,7 @@ std::shared_ptr<tesseract_planning::TrajOptDefaultCompositeProfile> createTrajOp
 
   profile->contact_test_type = tesseract_collision::ContactTestType::CLOSEST;
 
-  profile->collision_cost_config.enabled = true;
+  profile->collision_cost_config.enabled = false;
```

Comment:
I think this is another place where we actually want both the cost and constraints to be on. Otherwise, the constraint is probably going to fluctuate between being satisfied and unsatisfied